lib/suppressions.cpp

```diff
@@ -274,6 +274,23 @@ bool Suppressions::isSuppressed(const Suppressions::ErrorMessage &errmsg)
     return false;
 }
 
+void Suppressions::dump(std::ostream & out)
+{
+	out << "  <suppressions>" << std::endl;
+	for (std::list<Suppression>::const_iterator it = _suppressions.begin(); it != _suppressions.end(); ++it) {
```

Comment:
A few days ago we started allowing ranged for loops in Cppcheck source code so I would suggest:

    for (const Suppression &suppression : _suppressions)